index.html

```diff
@@ -41,5 +41,181 @@ <h1>I’m Marcelo
         </ul>
     </div>
 </section>
+<!---->
+<section class="work">
+    <!-- project-1 -->
+    <div class="project-1 projects">
+        <div class="card">
+            <span><img src="/imagenes/project_1.png" alt="project_1"></span>
+            <span><h2>Tonic</h2></span>
+            <div class="compn-lan">
+                <p>CANOPY</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="card-body">
+                <p>A daily selection of privately
+                     personalized reads; no accounts or 
+                     sign-ups required.</p>
+            </div>
+            <div class="technology">
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+            </div>
+            <button class="button">See Project</button>
```

Comment:
Nice! the button is designed precisely as the one in the Figma template. But according to the project requirements, adding the hover, active and disabled effects to your button is essential.

![image](https://user-images.githubusercontent.com/34779928/192348188-c2953a53-e509-410d-92c7-b2f4f4fc30ea.png)

Reply:
Where can I check this information? To add the hover I mean

Reply:
How do I add the hover ?? I don't understand why the last code reviewer didn't let me know about this... Its always a different change with the different code reviewer.